index.html

```diff
@@ -13,28 +13,149 @@
-        </div>
-       </header>
-       <section class="main-section">
-        <h1>I’m Naji
-         Glad to see you!</h1>
-        <p class="description">
-         I’m a software developer! I can help you build a product , feature or website Look through some of my work and
-         experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-        <h5>LET'S CONNECT</h5>
-        <ul class="links">
-         <li><a title="linkdin" href="https://www.linkedin.com/in/naji-ali-55b5421b3/"><i class="fa-brands fa-linkedin fa-2x"></i></a></li>
-         <li><a title="github" href="https://github.com/naji-97"><img src="./assest/icons/Vector.svg" alt="" srcset=""></a></li>
-         <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (3).svg" alt="" srcset=""></a></li>
-         <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg" alt="" srcset=""></a></li>
-         <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg" alt="" srcset=""></a></li>
-        </ul>
-       </section>
+ <header>
+  <div class="navbar">
+   <a class="logo" href="#"> My logo</a>
+   <div class="navbar"><i class="fa-solid fa-bars"></i></div>
+  </div>
+ </header>
+ <section class="main-section">
+  <h1>I’m Naji
+   Glad to see you!</h1>
+  <p class="description">
+   I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+   experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+  </p>
+  <h5>LET'S CONNECT</h5>
+  <ul class="links">
+   <li><a title="linkdin" href="https://www.linkedin.com/in/naji-ali-55b5421b3/"><i class="fa-brands fa-linkedin fa-2x"></i></a></li>
+   <li><a title="github" href="https://github.com/naji-97"><img src="./assest/icons/Vector.svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (3).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg" alt="" srcset=""></a></li>
+  </ul>
+ </section>
+
+  
+
+ <section class="projects">
+  <div class="project">
+  <img src="./assest/img/Snapshoot Portfolio.png" alt="">
+  <h2>Tonic</h2>
+  <ul class="toni">
+   <h5>CANOPY</h5>
+   <li> <span></span> Back End Dev</li>
+   <li> <span></span> 2022</li>
+  </ul>
+  <p class="decription">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+   <ul class="language">
+    <li>html</li>
+    <li>css</li>
+    <li>js</li>
+   </ul>
+   <div class="see-project">
+    <p><a href="#">See project</a></p>
+   </div>
```

Comment:
Kindly use a button tag for the see project and style the various states (hover, active and disabled (where necessary)) as required in the project to make it interactive.

see here...


<img width="334" alt="btn states" src="https://user-images.githubusercontent.com/84930355/205450303-74563667-dbd6-4b12-877e-d716ff223d2a.png">